lib/src/main/java/com/diffplug/spotless/npm/EslintFormatterStep.java

```diff
@@ -0,0 +1,273 @@
+/*
+ * Copyright 2016-2023 DiffPlug
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.diffplug.spotless.npm;
+
+import static java.util.Objects.requireNonNull;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.Serializable;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Objects;
+import java.util.TreeMap;
+import java.util.function.Predicate;
+
+import javax.annotation.Nonnull;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.diffplug.spotless.FormatterFunc;
+import com.diffplug.spotless.FormatterFunc.Closeable;
+import com.diffplug.spotless.FormatterStep;
+import com.diffplug.spotless.Provisioner;
+import com.diffplug.spotless.ThrowingEx;
+import com.diffplug.spotless.npm.EslintRestService.FormatOption;
+
+public class EslintFormatterStep {
+
+	private static final Logger logger = LoggerFactory.getLogger(EslintFormatterStep.class);
+
+	public static final String NAME = "eslint-format";
+
+	public static final String DEFAULT_ESLINT_VERSION = "^8.30.0";
```

Comment:
These are version numbers a user gets when running the configuration wizard `npm init @eslint/config`. I am unsure if we want the versions to be fixed (e.g. `8.30.0`) or if we want to follow the version numbers the configuration wizard generates (e.g. `^8.30.0`). What is your opinion on that?

Reply:
Author's choice :)